### PR TITLE
Covering observation wallet and modify translation

### PR DESCRIPTION
**Pull request by peterQin0826**
## What does this implement/fix? Explain your changes.
 覆盖观察钱包Feature
## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
OneKeyHQ/TaskHub#618，OneKeyHQ/TaskHub#599

## Pull request type
 
_Put an `x` in the boxes that apply_
- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
…

## Any other comments?
…

### CODE REVIEW

**Comment by peterQin0826** [comment]
@shuaiLiWang ,帮忙Review下

File: android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -153,7 +153,7 @@
     <string name="transaction_setting">Transaction Settings</string>
     <string name="service_online">Online Service</string>
     <string name="about">About (Terms of Agreement)</string>
-    <string name="zh_simple">中文（简体）</string>
+    <string name="zh_simple">Chinese (Simplified)</string>
```
Comment (QuincySx) [inline android/app/src/main/res/values/strings.xml:156]: 这是否需要修改

File: android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/HdWalletDetailActivity.java [hunk 2]
```diff
@@ -126,17 +129,26 @@ private void inits() {
...
         } else if (StringConstant.Btc_Standard.equals(showWalletType) || StringConstant.Btc_Private_Standard.equals(showWalletType)) {
             //Independent Wallet
             linHdWalletShow.setVisibility(View.GONE);
             linSingleShow.setVisibility(View.VISIBLE);
             textHdWallet.setText(getString(R.string.single_wallet));
+            if (StringConstant.Btc_Private_Standard.equals(showWalletType)) {
+                exportWordLayout.setVisibility(View.GONE);
+                exportPrivateLayout.setVisibility(View.VISIBLE);
+            } else {
+                exportPrivateLayout.setVisibility(View.GONE);
```
Comment (QuincySx) [inline android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/HdWalletDetailActivity.java:149]: 中间还有根线需要隐藏。

File: android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/WalletListActivity.java [hunk 3]
```diff
@@ -112,7 +112,7 @@ public void initData() {
         getHomeWalletList();
     }
 
-    @SingleClick(value = 6000L)
+    @SingleClick()
```
Comment (QuincySx) [inline android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/WalletListActivity.java:115]: 这个地方是李琰怕硬件反应不过来做的。拿硬件狂点一下，如果没问题 () 也删了吧

File: android/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (peterQin0826) [reply android/app/src/main/res/values/strings.xml:156]: 需要的

File: android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/WalletListActivity.java (hunk 3 above)
Comment (peterQin0826) [reply android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/WalletListActivity.java:115]: OK

File: android/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (QuincySx) [reply android/app/src/main/res/values/strings.xml:156]: 可以的，看产品需求吧。我是觉得这个地方放中文体验更好。

**Comment by peterQin0826** [comment]
@rayston92 翻译文件更新了

File: android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/HdWalletDetailActivity.java (hunk 2 above)
Comment (peterQin0826) [reply android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/HdWalletDetailActivity.java:149]: 我对比了设计文件，单独一条的是没有分割线，所以我就把之前的线删除了

**Review by QuincySx — APPROVED** [review]

**Pull request merged**